mail_activity/static/src/less/systray.less

```diff
@@ -0,0 +1,41 @@
+// Navbar icon and dropdown
+.o_mail_navbar_item {
```

Comment:
Can you improve the layout of the activitymenu items, please?
![image](https://user-images.githubusercontent.com/10028499/41641749-9baaef64-7466-11e8-9b06-642c4e573d7a.png)

Reply:
Looks better now :)

![image](https://user-images.githubusercontent.com/7683926/41669747-d3216436-74b2-11e8-9808-72e75dfad251.png)